src/Sulu/Bundle/AdminBundle/Resources/js/components/ColumnList/Column.js

```diff
@@ -4,26 +4,24 @@ import React from 'react';
 import classNames from 'classnames';
 import Loader from '../Loader';
 import Item from './Item';
-import Toolbar from './Toolbar';
-import type {ItemButtonConfig, ToolbarItemConfig} from './types';
+import type {ItemButtonConfig} from './types';
 import columnStyles from './column.scss';
 
-type Props = {
-    active: boolean,
+type Props = {|
     buttons?: Array<ItemButtonConfig>,
     children?: ChildrenArray<Element<typeof Item>>,
     index?: number,
     loading: boolean,
     onActive?: (index?: number) => void,
     onItemClick?: (id: string | number) => void,
-    toolbarItems: Array<ToolbarItemConfig>,
-};
+    /** @ignore */
+    scrolling: boolean,
```

Comment:
It's a pity that we need this prop... We thought it should be possible to get rid of the last border by letting them collapse, but it didn't really work out...